RichClipboard.package/RCClippingsWindow.class/instance/createClearAllButton.st

```diff
@@ -0,0 +1,12 @@
+clear-all
+createClearAllButton
+	" Private "
+	| button |
+	
+	button := PluggableButtonMorphPlus on: self
+											getState: #clearAllButtonEnabled
+											action: #clearAllEntries
+											label: #clearAllButtonLabel.
+	button height: self class defaultBottomBarHeight - 4.
```

Comment:
Diese Einrückung sieht komisch aus. Ist das so gewollt?

Reply:
das liegt daran, dass GitHub Tabs 8 Zeichen breit rendered. In Squeak ist das nichtmal monospaced Font, also sind Leerzeichen da sogar noch dünner als ein normales Zeichen. Jeder Tab wird hier also 8 Zeichen breit angezeigt, wärend er in Squeak nur ungefähr als 2 Zeichen breit zu sehen ist. Das sollte in Squeak richtig eingerückt sein :P